pkgs/applications/misc/tuhi/default.nix

```
@@ -13,16 +13,16 @@
 }:
 
 python3Packages.buildPythonApplication rec {
-  pname = "tuhi";
-  version = "0.5";
+  name = "tuhi";
```

This should be `pname`
```suggestion
  pname = "tuhi";
```